fuse/fuse_test.go

```diff
@@ -480,3 +480,99 @@ func TestWithRoot(t *testing.T) {
 		require.NotNil(t, err)
 	})
 }
+
+// Benchmarks
+
+var (
+	BenchmarkDataSizes = []int64{
+		0,
+		1024, 2*1024, 16*1024, 64*1024, 128*1023,
+		1*1024*1024, 16*1024*1024,
+	}
+)
+
+func stageAndRead(b *testing.B, ctx context.Context, control *spawntest.Control, mount *mountInfo, label string, data []byte) {
+	size := len(data)
+	// stage data to catFS
+	catfsFilePath := fmt.Sprintf("%s_file_%d", label, size)
+	req := catfsPayload{Path: catfsFilePath, Data: data}
+	require.Nil(b, control.JSON("/catfsStage").Call(ctx, req, &nothing{}))
```

Comment:
I nowadays recommend to use ``require.NoError`` since it produces better error messages when something fails. There's many places where around ``brig`` where still ``require.Nil`` is used, so I won't mind if you change them when you modify something around there.

Reply:
Replaced accordingly  everywhere in fuse-test.go with 0fa89b70

I would say we will need similar global patch, but this is for another branch.

Reply:
Either that or just replace it along other modifications.